loadtest/store/memstore/store.go

```diff
@@ -53,8 +53,10 @@ func (s *MemStore) SetUser(user *model.User) error {
 	return nil
 }
 
-func (s *MemStore) Preferences() (*model.Preferences, error) {
-	return s.preferences, nil
+func (s *MemStore) Preferences() (model.Preferences, error) {
+	newPref := make(model.Preferences, len(*s.preferences))
```

Comment:
I'd like to add a check for `s.preferences != nil` so we don't run into troubles.